tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py

```diff
@@ -413,6 +413,24 @@ def set_custom_predict_fn(self, predict_fn):
       - For regression: A 1D list of numbers, with a regression score for each
         example being predicted.
 
+    Optionally, if attributions can be returned by the model with each
+    prediction, then this method can return a dict with the key 'predictions'
+    containing the preidctions result list described above, and with the key
```

Comment:
typo preidctions, this happens in a lot of places so mass replace should fix it.

Reply:
done